### PR TITLE
Handle bulk_create() with ignore_conflicts

### PR DESCRIPTION
**Pull request by Spooner**
Django 3 now calls bulk_create() with ignore_conflicts, so it needs to be handled. This seems a reasonably sensible way to manage it, I hope.

### CODE REVIEW

**Comment by DeanSherwin** [comment]
I have this problem. Breaks DRF PrimaryKeyRelatedField when `many=True`.  Any timeline for when this fix will be included in a release?

**Comment by mateuszdargacz** [comment]
Any update on that?

**Comment by JanMalte** [comment]
@bfirsh @shuckc @imomaliev is there anything needed to get this MR merged and create a new release?

**Comment by imomaliev** [comment]
CI stuck, could you repush this?

**Comment by JanMalte** [comment]
>  Build type disabled via repository settings (pull_request).

Seems like MR are disabled in the Travis repo settings:
https://travis-ci.org/github/bfirsh/django-ordered-model/requests

File: ordered_model/models.py [hunk 1]
```diff
@@ -88,7 +89,7 @@ def increase_order(self, **extra_kwargs):
             update_kwargs.update(extra_kwargs)
         return self.update(**update_kwargs)
 
-    def bulk_create(self, objs, batch_size=None):
+    def bulk_create(self, objs, batch_size=None, ignore_conflicts=False):
```
Comment (JanMalte) [inline ordered_model/models.py:92]: Would it be better to use `def bulk_create(self, *args, **kwargs):` as we are not using any of the arguments and pass all to the parent class?
There would be no version check be required, if we are not using specific arguments.

```
     def bulk_create(self, objs, *args, **kwargs): 
```

File: ordered_model/models.py [hunk 2]
```diff
@@ -109,7 +110,11 @@ def bulk_create(self, objs, batch_size=None):
         else:
             for order, obj in enumerate(objs, self.get_next_order()):
                 setattr(obj, order_field_name, order)
-        return super().bulk_create(objs, batch_size=batch_size)
+
```
Comment (JanMalte) [inline ordered_model/models.py:113]:
```
         return super().bulk_create(objs, *args, **kwargs) 
```

**Comment by JanMalte** [comment]
@Spooner Could you repush this PR? I'm not able to update the files in this PR.

**Comment by Thor--** [comment]
It looks like @Spooner dropped off the map, so I implemented those changes here  https://github.com/bfirsh/django-ordered-model/pull/235

**Comment by shuckc** [comment]
Thanks @Thor-- @Spooner and @JanMalte I have merged #235 to fix this.

**Pull request closed without merging**